CrossSolar/Controllers/AnalyticsController.cs

```diff
@@ -51,9 +51,39 @@ public async Task<IActionResult> Get([FromRoute]string panelId)
 
         // GET panel/XXXX1111YYYY2222/analytics/day
         [HttpGet("{panelId}/[controller]/day")]
-        public async Task<IActionResult> DayResults([FromRoute]string panelId)
+        public async Task<IActionResult> DayResults([FromRoute] string panelId)
         {
-            throw new NotImplementedException();
+            var panelLastDayData = await _analyticsRepository.Query()
```

Comment:
This cannot be right, we need data for actuall day. Unit test may be passing, but it only checks for OK result, but not if correct data are returned.